.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Thanks for submitting a pull request 🎉! Here are some tips for you:
+
+* If this is your first contribution, read "Cargo Contribution Guide":
+  https://doc.crates.io/contrib/
+* Run `cargo fmt --all` to format your code changes.
+* Small commits and pull requests are always preferable and easy to review.
+* If your idea is large and needs feedback from the community, read how:
+  https://doc.crates.io/contrib/process/#working-on-large-features
+* Cargo takes care of compatibility. Read our design principles:
+  https://doc.crates.io/contrib/design.html
+* When changing help text of cargo commands, follow the steps to generate docs:
+  https://github.com/rust-lang/cargo/tree/master/src/doc#building-the-man-pages
+* If your PR is not finished, set it as "draft" PR or add "WIP" in its title.
+* It's ok to use the CI resources to test your PR, but please don't abuse them.
+
+Please DELETE all these comments before submitting the PR.
```

Comment:
I don't think there's actually a need to do this in the sense that no one else ever really sees this HTML comment. If that's the case I think it's ok to remove this line. I know I at least never delete the default comment myself :)